src/utils/handleLikeUnlike.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import checkLogin from "./checkLogin";
+import toast from "./toast";
+
+const handleLikeVideo = async (videoData, tokenName, isLoggedIn, dispatchUser) => {
+    if (checkLogin(isLoggedIn)) {
+      try {
+        const res = await axios.post(
+          "/api/user/likes",
+          { video: videoData },
+          {
+            headers: {
+              authorization: localStorage.getItem(tokenName),
+            },
+          }
+        );
+        if (res.status === 200 || res.status === 201) {
+          const { likes } = res.data;
```

Comment:
you have to no need this line you can do on line number 8 like this.

const { data : {likes}, status }